docset/winserver2025-ps/hyper-v/New-VM.md

```diff
@@ -260,10 +284,33 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -GuestStateIsolationType
+
+Specifies the level of security for the virtual machine being created. Acceptable values are:
+
+- `TrustedLaunch`: Enables Trusted Launch security feature for the virtual machine.
+- `VBS`: Enables Virtualization Based Security (VBS) for the virtual machine.
+- `SNP`: Enables System and Network Protection (SNP) for the virtual machine.
```

Comment:
TrustedLaunch:  Enable guest state isolation and Trusted Launch security features...
VBS:  Enables Virtualization-based Security (VBS) isolation...
SNP:  Enables AMD Secure Encrypted Virtualization-Secure Nested Paging (SEV-SNP) isolation...
TDX:  Enables Intel Trust Domain Extensions (TDX) isolation...